pan3d/explorers/slice_explorer.py

```diff
@@ -0,0 +1,441 @@
+        self.renderer.SetActiveCamera(camera)
+        self.renderer.ResetCamera()
+        self.ctrl.view_update()
+
+    @change("data_var", "time_active", "dimval")
+    def on_data_change(self, **_):
+        dimval    = self.state.dimval
+        slice_dim = self.state.slice_dim
+        normal      = [0, 0, 0]
+        slice_i     = 0 if self.dims[slice_dim] == 'x' else 1 if self.dims[slice_dim] == 'y' else 2
+        normal[slice_i] = 1
+        
+        extents = list(self.extents.values())
+        origin = [
+            float(extents[0][0] + (extents[0][1] - extents[0][0]) / 2), 
+            float(extents[1][0] + (extents[1][1] - extents[1][0]) / 2), 
+            float(extents[2][0] + (extents[2][1] - extents[2][0]) / 2)
+        ] 
+        origin[slice_i] = dimval
+        
+        self.plane.SetOrigin(origin)
+        self.plane.SetNormal(normal)
+
+        self.cutter.SetInputData(self.t_slice)
+        self.cutter.SetCutFunction(self.plane)
+        self.cutter.Update()
+        output = self.cutter.GetOutput()
+        vrange = output.GetPointData().GetArray(self.state.data_var).GetRange()
+        self.slice_mapper.SetScalarRange(float(vrange[0]), float(vrange[1]))
+        self.state.varmin = float(vrange[0])
+        self.state.varmax = float(vrange[1])
+        self.sbar_actor.SetLookupTable(self.slice_mapper.GetLookupTable())
+
+        if self.state.view_mode == "2D":
+            self.on_view_mode_change("2D")
+            self.renderer.ResetCamera()
+
+        self.ctrl.view_update()
+
+    @change("outline", "tdata")
+    def on_rep_change(self, outline, tdata, **_):
+        if outline: 
+            self.renderer.AddActor(self.outline_actor)
+        else:
+            self.renderer.RemoveActor(self.outline_actor)
+        if tdata:
+            self.renderer.AddActor(self.data_actor)
+        else:
+            self.renderer.RemoveActor(self.data_actor)
+        self.ctrl.view_update()
+
+    @change("varmin", "varmax")
+    def on_scalar_change(self, varmin, varmax, **_):
+        self.slice_mapper.SetScalarRange(float(varmin), float(varmax))
+        self.sbar_actor.SetLookupTable(self.slice_mapper.GetLookupTable())
+        self.ctrl.view_update()
+        
+    @property
+    def levcoords(self):
+        return self.builder.dataset.coords["level"].to_numpy()
+
+    @property
+    def timecoords(self):
+        return self.builder.dataset.coords["month"].to_numpy()
+    
+    def UpdateSlicer(self, dim):
+        ext = self.extents[dim]
+        self.state.dimval = float(ext[0] + (ext[1] - ext[0]) / 2)
+        self.state.dimmin = float(ext[0])
+        self.state.dimmax = float(ext[1])
+
+    def start(self, **kwargs):
+        self.ui.server.start(**kwargs)
+
+    @property
+    def ui(self):
+        if self._ui:
+            return self._ui
+
+        style = dict(density="compact", hide_details=True)
+
+        with SinglePageWithDrawerLayout(self.server, full_height=True) as layout:
```

Comment:
Does the `full_height=True` remove the slider in your view? If not we need to add a client.Style("html, body {  overflow: hidden; }")

Reply:
I tried setting the property on the client, but still see the scroll bar on the right.

Reply:
ok then add the `client.Style()` unless your layout leak outside the main content, in which case, you should fix that part first.

Reply:
@ayenpure Please make a formal issue for this so that we can move ahead on this PR.

Side note: this scrollbar does NOT show up in my firefox browser. (Some other unexpected things occur, but I'll detail later if needed).

Reply:
Done!